bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -38,6 +40,10 @@
 				<description>The IP address of the Go-eCharger</description>
 				<context>network-address</context>
 			</parameter>
+			<parameter name="apiVersion" type="integer" required="true">
```

Comment:
You can use attributes `min` and `max` is you want the configuration validated so only values 1 and 2 will be allowed.